user-guide/Advanced_Modules/Dashboards_app/Dashboard_Gateway_installation.md

```diff
@@ -67,3 +71,100 @@ There are two main reasons to consider a Dashboard Gateway setup:
+
+### Reverse proxy requirements
+
+- IIS 7 or higher with ASP.NET role service enabled
+
+- [URL Rewrite](https://www.iis.net/downloads/microsoft/url-rewrite)
+
+- [Application Request Routing](https://www.iis.net/downloads/microsoft/application-request-routing)
+
+### Configuration
+
+#### Enabling reverse proxy functionality
+
+As the reverse proxy functionality is disabled by default, first enable it by doing the following:
+
+1. Open IIS Manager on the Gateway server.
+
+1. In the tree view on the left, choose a server node and select the *Application Request Routing* feature.
+
+1. Select the *Enable Proxy* checkbox.
+
+1. All the other settings can be left on default.
+
+#### Creating rewrite rules for the Maps module
+
+To add a rewrite rule that will proxy any request to the DataMiner Maps module at `https://mydma/maps/`, as long as the requested URL path starts with "maps", do the following:
+
+1. Open IIS Manager on the Gateway server.
+
+1. In the tree view on the left, choose a web site node and select the *URL Rewrite* feature.
+
+1. Click *Add Rule(s)*, select *Blank rule*, and specify the following options:
+
+   - **Name**: e.g. "Maps module"
+
+   - **Pattern**: `^maps/(.*)`
+
+   - **Rewrite URL**: `https://mydma/maps/{R:1}`
+
+   - ***Stop processing of subsequent rules***: enabled
+
+1. Click *apply*.
+
+To add a rewrite rule that will proxy any request to the Maps API at `https://mydma/API/v0/maps.asmx`, as long as the requested URL path starts with "API/v0/maps.asmx", do the following:
+
+1. Open IIS Manager on the Gateway server.
+
+1. In the tree view on the left, choose a web site node and select the *URL Rewrite* feature.
+
+1. Click *Add Rule(s)*, select *Blank rule*, and specify the following options:
+
+   - **Name**: e.g. "Maps API"
+
+   - **Pattern**: `^API/v0/maps.asmx/(.*)`
+
+   - **Rewrite URL**: `https://mydma/API/v0/maps.asmx/{R:1}`
+
+   - ***Stop processing of subsequent rules***: enabled
+
+1. Click *apply*.
+
+#### Creating rewrite rules for the VideoThumbnails web page
+
+To add a rewrite rule that will proxy any request to the VideoThumbnails web page at `https://mydma/videothumbnails/`, as long as the requested URL path starts with "videothumbnails", do the following:
+
+1. Open IIS Manager on the Gateway server.
+
+1. In the tree view on the left, choose a web site node and select the *URL Rewrite* feature.
+
+1. Click *Add Rule(s)*, select *Blank rule*, and specify the following options:
+
+   - **Name**: e.g. "VideoThumbnails"
+
+   - **Pattern**: `^videothumbnails/(.*)`
+
+   - **Rewrite URL**: `https://mydma/videothumbnails/{R:1}`
+
+   - ***Stop processing of subsequent rules***: enabled
+
+1. Click *apply*.
+
+### Testing the reverse proxy functionality
+
+To test whether the reverse proxy is working properly, do the following:
+
+1. Enter `https://gateway/maps/maps.aspx?config=yourmapconfig` or `https://gateway/videothumbnails/video.htm` in your browser's address bar.
```

Comment:
I have reworded this step. Is this still OK content-wise?

Reply:
Yes! LGTM